dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/validation/ValidationEnhancer.java

```diff
@@ -0,0 +1,56 @@
+package edu.harvard.iq.dataverse.validation;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import edu.harvard.iq.dataverse.persistence.dataset.DatasetFieldType;
+import edu.harvard.iq.dataverse.validation.field.FieldValidator;
+import edu.harvard.iq.dataverse.validation.field.ValidationDescriptor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Collections;
+import java.util.Map;
+
+public class ValidationEnhancer {
```

Comment:
Can we add some comments in this class? Some explanation why we even need it would be helpful. Meaning that we want to enhance validation for things that formally are not dataset fields.